climada/engine/unsequa/test/test_unsequa.py

```diff
@@ -36,6 +36,8 @@
 from climada.hazard import Hazard
 from climada.engine import ImpactCalc
 from climada.engine.unsequa import InputVar, CalcImpact, UncOutput, CalcCostBenefit, CalcDeltaImpact
+from climada.engine.unsequa.calc_base import LOGGER as ILOG
```

Comment:
Any particular reason for renaming the logger? Why not just use `LOGGER` ?

Reply:
No, no reason, it was just blind copying of the code I saw on stackoverflow, I'll change that!